usaspending_api/download/v2/base_download_viewset.py

```diff
@@ -85,6 +115,17 @@ def process_request(self, download_job: DownloadJob):
             queue = get_sqs_queue(queue_name=settings.BULK_DOWNLOAD_SQS_QUEUE_NAME)
             queue.send_message(MessageBody=str(download_job.download_job_id))
 
+    def process_account_download_in_spark(self, download_job: DownloadJob):
+        """
+        Process File C downloads through spark instead of sqs for better performance
+        """
+        spark_jobs = SparkJobs(LocalStrategy()) if IS_LOCAL else SparkJobs(DatabricksStrategy())
+        spark_jobs.start(
+            job_name="download_delta_table-award_search",
```

Comment:
We never decided on a name for the download, but I would recommend changing this to `api_download-accounts`. Open to other ideas but my thought process is that we want to distinguish this as a download generated by the API, and it is an "accounts" download.